src/main/java/com/github/alexcojocaru/mojo/elasticsearch/RunElasticSearchNodeMojo.java

```diff
@@ -0,0 +1,66 @@
+package com.github.alexcojocaru.mojo.elasticsearch;
+
+import org.apache.maven.plugin.MojoExecutionException;
+
+import java.io.File;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Starts a local instance of ElasticSearch indefinitely.
+ * In order to kill it a CTRL+C has to be made.
+ *
+ * @author gfernandes
+ * @goal run
+ * @execute phase="compile"
+ * @requiresDependencyResolution runtime
+ */
+public class RunElasticSearchNodeMojo extends AbstractStartElasticsearchNodeMojo {
```

Comment:
This is one is the main reason for the pull request. I needed to have a goal that runs the ES local instance in order to be used as the target ES from my angular-js front-end during development and I wanted to do it with maven. Something similar to what http://tomcat.apache.org/maven-plugin-2.0/tomcat7-maven-plugin/ provides.
At first I thought that it will also be good to keep it running in a fork, it worked fine but didn't find a way to "shutdown" the ES instance due to the shutdown rest end-point is deprecated. It would be good to find a way to do this so another mojo for shutdown goal could be implemented.